Formula/hg-fast-export.rb

```diff
@@ -0,0 +1,19 @@
+class HgFastExport < Formula
+  desc "Mercurial to git converter using git-fast-import"
+  homepage "https://packages.debian.org/source/sid/hg-fast-export"
+  url "http://http.debian.net/debian/pool/main/h/hg-fast-export/hg-fast-export_20140308.orig.tar.gz"
```

Comment:
Is Debian the upstream for this? Is this a stable version tagged by upstream or just a Debian tarball versioned by date?